docs/installation.rst

```diff
@@ -4,14 +4,14 @@ Installation
 
 Minimal Requirements
 ------------
-* Python 3.10 to Python 3.12. We recommend Python 3.10.
+* Python 3.10 to Python 3.12. We recommend Python 3.12.
 * Microsoft Build Tools for Visual Studio 2019 (only Windows)
 * 6GB RAM (minimum 1GB *free* RAM during the session)
 * Intel i5 or higher (2x2.5GHz) CPU
 
 Recommended Requirements
 ------------
-* Python 3.10
+* Python 3.7 to Python 3.12
```

Comment:
"Recommended" should be only 3.12, I think (or 3.10-3.12, but if the minimal requirements are stricter than the recommended ones, that's a bit odd)